generators/chipyard/src/main/scala/config/IbexConfigs.scala

```diff
@@ -0,0 +1,14 @@
+package chipyard
+
+import chisel3._
+
+import freechips.rocketchip.config.{Config}
+
+// ---------------------
+// Ibex Configs
+// ---------------------
+
+class IbexConfig extends Config(
+  new chipyard.config.WithBootROM ++               // Ibex reset vector is at 0x80

```

Comment:
I also this breaks all other cores since you need to change the hang address and the base address of the bootrom.

See: https://github.com/ucb-bar/testchipip/pull/136#issuecomment-918468095
Params you need to change in the `WithBootROM` fragment: https://github.com/chipsalliance/rocket-chip/blob/c4c43134e6cb604ecc329dd8e32614b8a976bae7/src/main/scala/devices/tilelink/BootROM.scala#L17-L19

By changing the WithBootROM, you might also need to fix up some of the Ibex core code as well so that it parses the hang or address params properly.